.github/workflows/binary-builds.yml

```diff
@@ -37,274 +37,202 @@ concurrency:
 permissions: {}
 
 jobs:
-  musl-builds:
+  binaries:
     if: github.repository == 'CycloneDX/cdxgen'
     strategy:
       fail-fast: false
       matrix:
-        os: [ 'ubuntu-22.04', 'ubuntu-22.04-arm' ]
+        os: [ linux, macos, windows ]
+        arch: [ amd64, arm64]
+        format: [ musl, sae ]
+        node: [ 24 ]
+        alpine: [ 3.22 ]
```

Comment:
Let's keep 3.21 and ubuntu 22.04. The idea is to use the lowest glibc and musl versions so that the binaries can have broader compatibility. Otherwise, the binaries will work only on the latest OS.